src/issuer.ts

```diff
@@ -49,6 +49,7 @@ export class Issuer {
     sdJWTPayload: CreateSDJWTPayload,
     sdVCClaimsDisclosureFrame: DisclosureFrame = {},
     saltGenerator?: SaltGenerator,
+    header?: Record<string, any>,
```

Comment:
1. **Typings**: It would be beneficial to add better typings for the `sdJWTHeader`. This will improve code readability and maintainability.

2. **Method Signature**: The current method signature is cumbersome due to the large number of optional parameters. Consider refactoring this to use an options object pattern. This will make the function easier to use and understand.

Here's a proposed refactor:

```typescript
// Add new CreateSignedJWTOpts interface
export interface CreateSignedJWTOpts {
  vcClaims: VCClaims;
  sdJWTPayload: CreateSDJWTPayload;
  sdVCClaimsDisclosureFrame?: DisclosureFrame;
  saltGenerator?: SaltGenerator;
  sdJWTHeader?: Omit<JWTHeaderParameters, 'typ' | 'alg'>;
}

// Deprecate existing method signature 
/**
 * @deprecated This method will be removed in the next version. Use `createSignedVCSDJWT` instead.
 */
async createVCSDJWT(
  vcClaims: VCClaims,
  sdJWTPayload: CreateSDJWTPayload,
  sdVCClaimsDisclosureFrame: DisclosureFrame = {},
  saltGenerator?: SaltGenerator,
  sdJWTHeader?: Omit<JWTHeaderParameters, 'typ' | 'alg'>,
): Promise<JWT> {
  return this.createSignedVCSDJWT({
    vcClaims,
    sdJWTPayload,
    sdVCClaimsDisclosureFrame,
    saltGenerator,
    sdJWTHeader,
  });
}

// Create new method signature
/**
 * Creates a signed VC SD-JWT.
 */
async createSignedVCSDJWT(opts: CreateSignedJWTOpts): Promise<JWT> {
  const { vcClaims, sdJWTPayload, sdVCClaimsDisclosureFrame = {}, saltGenerator, sdJWTHeader } = opts;

  // Rest of the code
  ...
}